src/system/execute.py

```diff
@@ -20,7 +20,7 @@ def execute(command: str, dir: str, capture: bool = True, raise_on_failure: bool
     :returns a tuple containing the exit code, stdout, and stderr.
     """
     logging.info(f'Executing "{command}" in {dir}')
-    result = subprocess.run(command, cwd=dir, shell=True, capture_output=capture, text=True)
+    result = subprocess.run(command, cwd=dir, shell=True, capture_output=capture, text=True, encoding='utf-8')
```

Comment:
Are we able to set the encoding globally instead of manually adding to each subprocess run?

Reply:
I think according to my research, the encoding would be default depending on the platform. Not sure whether we could set it globally somewhere.

Reply:
So does it affect the behavior on Linux?

Reply:
I don't think so. The default encoding for deb/linux should be `UTF-8` but not for windows.

Reply:
Thanks.